src/click/core.py

```diff
@@ -2337,7 +2337,15 @@ def convert(value: t.Any) -> t.Any:  # tuple[t.Any, ...]
         return convert(value)
 
     def value_is_missing(self, value: t.Any) -> bool:
-        if value is None:
+        """
```

Comment:
Any methods that were not previously documented did not show up in the API docs, so were not considered public. Add `:meta private:` at the bottom of each of these method's docstrings, until we determine what we want to make public.